code/aepcomposeui/build.gradle.kts

```diff
@@ -0,0 +1,35 @@
+plugins {
+    id("aep-library")
+}
+
+val mavenCoreVersion: String by project
+val aepComposeUiModuleName: String by project
+val aepComposeUiVersion: String by project
+val aepComposeUiMavenRepoName: String by project
+val aepComposeUiMavenRepoDescription: String by project
+
+aepLibrary {
+    namespace = "com.adobe.marketing.mobile.aepcomposeui"
+
+    moduleName = aepComposeUiModuleName
+    moduleVersion = aepComposeUiVersion
+    enableSpotless = true
+    enableCheckStyle = true
+    enableDokkaDoc = true
+
+    publishing {
+        mavenRepoName = aepComposeUiMavenRepoName
+        mavenRepoDescription = aepComposeUiMavenRepoDescription
+        gitRepoName = "aepsdk-messsaging-android"
+        addCoreDependency(mavenCoreVersion)
+    }
+}
+
+dependencies {
+    api(project(":aepuitemplates"))
+    implementation("com.adobe.marketing.mobile:core:$mavenCoreVersion")
```

Comment:
Why do we need Core SDK dependency here? Are there any utilities we're using from Core?

Reply:
I believe we need this for NetworkService. This will become implicit when we move this to a Messaging sub package.

Reply:
I added this to get dependency on compose in submodule